src/services/telegram/app.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import cherrypy
+
+from common.WIOTRestApp import *
+from common.SettingsManager import *
+from common.SettingsNode import *
+from common.RESTBase import RESTBase
+from common.CatalogRequest import CatalogRequest
+from TelMan import MyBot
+
+class TelegramAdaptorAPI(RESTBase):
+
+    def __init__(self, upperRESTSrvcApp, settings: SettingsNode, telegramkey) -> None:
+        super().__init__(upperRESTSrvcApp, 0)
+        self._catreq = CatalogRequest(self.logger, settings)
+        self._bot = MyBot(telegramkey,self.logger,self._catreq)
+
+    @cherrypy.tools.json_out()
+    def GET(self, *path, **args):
+
+        if len(path)>0 and path[0] == "sendMessage":
+            try:
+                if not "text" in args: # check correct HTTP argument
+                    cherrypy.response.status = 400
+                    return self.asjson_error("Missing text argument")
+                    self._bot.bot.sendMessage(self._bot.chat_ID,args["text"])
+            except Exception as e:
+                self.logger.error(f"Error occurred in sending message: {str(e)}") # exception rised if is sent a message
+                cherrypy.response.status = 500 #Server side error                   before the user send a first message 
+                return self.asjson_error(f"Server error")                         # on chat ( chat_id is retrived)
```

Comment:
This is due to an update 

```suggestion
                return self.asjson_error(f"Server error", 500)
```